internal/plan/plan.go

```diff
@@ -83,7 +83,7 @@ func (e *FormatError) Error() string {
 	return e.Message
 }
 
-// CombineLayers combines the given layers into a Plan, with the later layers
+// CombineLayers combines the given layers into a plan, with the later layers
```

Comment:
`with the later layers layers`

Maybe consider revising?

Reply:
Thanks, updated.